mmdet/ops/masked_conv/modules/masked_conv.py

```diff
@@ -0,0 +1,29 @@
+import torch.nn as nn
+from ..functions.masked_conv import masked_conv2d
+
+
+class MaskedConv2d(nn.Conv2d):
+    """A MaskedConv2d which inherits the official Conv2d.
+    The masked forward doesn't implement the backward function and only
+    supports the stride parameter to be 1 currently.
+    """  # noqa: W605
```

Comment:
Example of python docstring: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html